File: android/app/src/main/java/com/hedvig/app/ServiceContributorModule.java
package com.hedvig.app;

import com.hedvig.android.owldroid.service.PushNotificationService;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
abstract class ServiceContributorModule {
    @ContributesAndroidInjector
    abstract PushNotificationService contributePushNotificationService();
}

File: android/app/src/main/java/com/hedvig/app/ServiceContributorModule.kt
package com.hedvig.app

import com.hedvig.android.owldroid.service.PushNotificationService
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
internal abstract class ServiceContributorModule {
    @ContributesAndroidInjector
    internal abstract fun contributePushNotificationService(): PushNotificationService
}
